### ISSUE TITLE
Training videos. Missing test_video.py?

### ISSUE THREAD

**Issue by RKelln** [vsitzmann/siren#42]
Trained up a a custom video but then realized there was no equivalent to `test_audio.py`. Is that just an oversight, as there is an example video on the project page?

Also note that there seems to be missing support for `checkpoint_path` in `train_video.py`, something like this can be added:
```python
if opt.checkpoint_path is not None:
    model.load_state_dict(torch.load(opt.checkpoint_path))
```
In my training the loss started increasing after hitting a minimum around 20000. Is that what you observed?
